helpers/plugins.js

```diff
@@ -1,11 +1,8 @@
 const pluginSets = new Map();
-
-export function getPlugin(setKey, pluginKey) {
-	const pluginSet = pluginSets.get(setKey);
-	return pluginSet?.plugins.find(plugin => plugin.options.key === pluginKey)?.plugin;
-}
+const pluginsRequested = new Set();
 
 export function getPlugins(setKey) {
+	if (!pluginsRequested.has(setKey)) pluginsRequested.add(setKey);
```

Comment:
Another option would be to set something like a `requested = true` on the set, similar to `requiresSorting`.

Reply:
Yeah, I originally considered that but felt wrong because `getPlugins` would need to add to the `pluginSets` for the case where there are no registrations. But now that I think about it more, tracking the bit separately isn't really saving anything, whether the plugin Set key is valid or not. I think I will switch it.